tests/unit/malware/test_checks.py

```diff
@@ -33,7 +33,7 @@ def test_checks_subclass_base():
         cls.__name__: cls for cls in MalwareCheckBase.__subclasses__()
     }
 
-    assert len(all_checks) == len(subclasses_of_malware_base)
+    assert len(all_checks) <= len(subclasses_of_malware_base)
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏I'm not sure this is the desirable fix - since we currently do only have 4 real subclasses, at least until the `test_base_prepare_nonfile_hooked` creates one later on. Would it be better to update the selector to reject `FakeProjectCheck` from the list if found, or something else like removing the subclass? 

For reference, here's what `subclasses_of_malware_base` looks like when the order is flipped:

```python
{'ExampleHookedCheck': <class 'tests.common.checks.hooked.ExampleHookedCheck'>,
 'ExampleScheduledCheck': <class 'tests.common.checks.scheduled.ExampleScheduledCheck'>,
 'FakeProjectCheck': <class '__main__.FakeProjectCheck'>,
 'PackageTurnoverCheck': <class 'warehouse.malware.checks.package_turnover.check.PackageTurnoverCheck'>,
 'SetupPatternCheck': <class 'warehouse.malware.checks.setup_patterns.check.SetupPatternCheck'>}
```

Reply:
In commit f36294c I tried the approach of cleaning up `FakeProjectCheck` after it is used for the test. Would that work alright or is one of the alternative approaches better?